tests/sles4sap/publiccloud/qesap_prevalidate.pm

```diff
@@ -55,7 +55,14 @@ sub run {
     }
 
     # Check cluster for overall readiness (nodes online, in sync and crm output contains no failed resources)
-    $self->wait_for_cluster(wait_time => 60, max_retries => 10);
+    # Make sure we're running an HA test and that instance in $self->{my_instance} is a hana node first
+    if ($ha_enabled) {
```

Comment:
From lines 58 untill the end of this function we have:
```
if $ha_enabled do sth
return if $ha_enabled
implicit if $ha_enabled do sth
implicit return
```
We should put everything in one if $ha_enabled conditional.

Reply:
You're right. Refactored that part.